2023-03_sprint/sprint_parties/contributors.md

```diff
@@ -111,3 +111,16 @@ Brooklyn, New York
 [{fab}`github`](https://github.com/vsaitejaswie)
 :::
 ::::
+
+:::{grid-item}
+
+<img alt="Image for contributor template" src="https://raw.githubusercontent.com/data-umbrella/data-umbrella.github.io/main/images/logo/umb-trans.png" class="rounded-circle">
+
```

Comment:
The `src` link to image needs to be updated.
This is the preview:
https://pymc-data-umbrella--224.org.readthedocs.build/en/224/2023-03_sprint/sprint_parties/contributors.html

If you prefer not to have a photo, try this one:
```
src="../../_static/contributors/blank_person.jpg"
```

Reply:
@reshamas I have updated the image url